sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/PropertyToAutoProperty.CSharp9.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+public class MyAttribute : Attribute { }
+
+public record Record
+{
+    private int field1;
+    private string field2;
+    private string field3;
+
+    [My()]
```

Comment:
what is the attribute testing?

Reply:
This is a double check that we don't raise for fields which have attributes.